pkg/lib/strings/operations.go

```diff
@@ -249,3 +249,25 @@ func RemoveDuplicates(strs []string, ignoreRegex *regexp.Regexp) []string {
 
 	return result
 }
+
+// SliceContains checks if a string slice contains a target string
+func SliceContains(strSlice []string, target string) bool {
+	for _, item := range strSlice {
+		if item == target {
+			return true
+		}
+	}
+	return false
+}
+
+// RemoveFromSlice removes a target string from a string slice if it exists
+func RemoveFromSlice(strSlice []string, target string) []string {
+	var result []string
+	for _, item := range strSlice {
+		if item == target {
+			continue
+		}
+		result = append(result, item)
+	}
+	return result
+}
```

Comment:
nit: We have `pkg/lib/slices/strings.go`, which includes `slices.HasString()` (feel free rename it). Also, perhaps `RemoveFromSlice()` should be moved there as well (we could leave the name as is, or match e.g. `slices.HasString()` so it would be `slices.RemoveString()`).